addon/wolfcrypt/main.cpp

```diff
@@ -28,11 +28,22 @@
 #include "./h/ecc.h"
 #include "./h/pbkdf2.h"
 #include "./h/pkcs7.h"
+#include "./h/pkcs12.h"
 
 using namespace Napi;
 
 Napi::Object Init(Napi::Env env, Napi::Object exports)
 {
+#ifdef HAVE_FIPS
+#ifdef WC_RNG_SEED_CB
+    wc_SetSeed_Cb(wc_GenerateSeed);
```

Comment:
Shouldn't HAVE_FIPS also set the FIPS callback so detecting/solving FIPS errors is easier?

```
static void myFipsCb(int ok, int err, const char* hash)
{
    printf("in my Fips callback, ok = %d, err = %d\n", ok, err);
    printf("message = %s\n", wc_GetErrorString(err));
    printf("hash = %s\n", hash);

    if (err == WC_NO_ERR_TRACE(IN_CORE_FIPS_E)) {
        printf("In core integrity hash check failure, copy above hash\n");
        printf("into verifyCore[] in fips_test.c and rebuild\n");
    }
}

wolfCrypt_SetCb_fips(myFipsCb);
```